cli/productutil.py

```diff
@@ -82,6 +82,30 @@ def parse_dependency_str(token):
     return parse_dependency_str
 
 
+def rescan_product_images(
```

Comment:
could the scanning mode not be an argument (so that no new function would need to be defined)?